_posts/2023-07-19-introducing-extensions-for-opensearch.md

```diff
@@ -0,0 +1,127 @@
+---
+layout: post
+title:  "Introducing extensions for OpenSearch"
+authors:
+  - vemsarat
+  - widdisd
+  - kazabdu
+  - dblock
+  - dagney
+  - minalsha
+  - kolchfa
+date: 2023-07-19
```

Comment:
We should release it a day after OpenSearch 2.9 release (on 24th) i.e. 25th. We will be releasing SDK repo on 25th.

Reply:
Changed the date to 7/25